build.sh

```diff
@@ -27,15 +42,40 @@ create_dummy_ksp () {
     fi
     
     # TODO: Manual hack, a better way to handle this kind of identifiers may be needed.
-    if [ "$KSP_VERSION" == "0.90" ]
-    then
+    case $KSP_VERSION in
+    "0.90")
+        echo "Overiding '0.90' with '0.90.0'"
         KSP_VERSION="0.90.0"
-    fi
+        ;;
+    "1.0")
+        echo "Overiding '1.0.0' with '1.0.0'"
```

Comment:
I really need to configure a spell checker pre-commit hook! Case statements use fairly simple matching so that won't match on 1.0.2. What we can probably do is set it to the default latest version.